.ci/pytorch/build.sh

```diff
@@ -159,6 +159,12 @@ if [[ "$BUILD_ENVIRONMENT" == *cuda* && -z "$TORCH_CUDA_ARCH_LIST" ]]; then
   exit 1
 fi
 
+# We only build FlashAttention files for CUDA 8.0, and they require large amounts of
+# memory to build and will OOM
+if [[ "$BUILD_ENVIRONMENT" == *cuda* ]] && [[ "$TORCH_CUDA_ARCH_LIST" == *"8.6"* ]]; then
+  export MAX_JOBS=2
```

Comment:
Try settings this to 3 again

Reply:
Please feel free to cut us an issue about this so we could look into this later to figure out if there is a better way to set it without slowing down the build.

Reply:
https://github.com/pytorch/pytorch/issues/108174

Reply:
Err, should this be limited only to debug builds?

Reply:
This is OOMing when building in with DEBUG=0